src/main/java/org/assertj/core/api/AbstractStringAssert.java

```diff
@@ -375,4 +376,26 @@ public SELF isEqualTo(String expectedStringTemplate, Object... args) {
   public SELF isEqualTo(String expected) {
     return super.isEqualTo(expected);
   }
+
+  /**
+   * Verifies that the actual value is equal to expected on their canonical form.
```

Comment:
Let's mention we use `Normalizer.Form.NFC` (add a link to it) so that it is clear what is being checked.